packages/ses/index.test-d.ts

```diff
@@ -51,37 +51,13 @@ const moduleMap = {
   'direct': 'redirect',
   'internal': {d: 40},
 };
-const moduleMapHook = (specifier: string) => {
```

Comment:
Removed because it wasn't relevant to the static types check. I'll restore it in an Ava test.

Reply:
Correction: removed because it was never run. (I was moving quickly and thought this was the file that I moved from Ava to tsd but this was always a static type test so the code body was irrelevant.)